Includes/survival-games.xml

```diff
@@ -57,15 +57,26 @@
         <region id="tier-2-chests"/>
     </all>
 </filters>
+<variables scope="player">
+    <variable id="tribute"/>
+</variables>
+<variables scope="match">
+    <variable id="tmp"/>
+    <array id="tp_points_x" size="${player-count}" />
+    <array id="tp_points_y" size="${player-count}" />
+    <array id="tp_points_z" size="${player-count}" />
+</variables>
 <actions>
     <trigger filter="starting-countdown" scope="player">
         <action scope="player">
-            <message text="`7`l[`6`lSurvival `e`lGames`7`l] `3The games have started!"/>
+            <sound key="ambient.weather.thunder" volume="0.2"/>
```

Comment:
```suggestion
            <sound key="${entity.lightning_bolt.thunder}" volume="0.2"/>
```

Use the [sound keys](https://github.com/Warzone/PublicMaps/blob/main/Includes/sound-keys.xml) (automatically loaded with the global.xml include) instead so it'll work across 1.8 and 1.21 PGM. Obviously I cannot test this, so try to keep this in mind for any other instance of sound actions.